meetings/2022-07-20.md

```diff
@@ -0,0 +1,85 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2022-07-20
+
+## Links
+
+* **Recording**:  <http://www.youtube.com/watch?v=wSz4LwljQ_8>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1261>
+* **Minutes Google Doc**: <https://github.com/nodejs/TSC/issues/1261>
+
+## Present
+
+* Antoine du Hamel @aduh95 (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Geoffrey Booth @GeoffreyBooth (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Darshan Sen @RaisinTen (TSC)
+* Richard Lau @richardlau (TSC)
+* Michaël Zasso @targos (TSC)
+* Tobias Nießen @tniessen (TSC)
```

Comment:
I was only able to attend the private section. I am not sure if that counts here :)